.env.example

```diff
@@ -0,0 +1,18 @@
+# Database keys
+DEVELOPMENT_DB_SECRET_KEY=
+TEST_DB_SECRET_KEY=
+SECRET_KEY_BASE=
+
+# Rails config
+RAILS_MAX_THREADS=
+
+# Database credentials
+DATAROCKETS_FINANCE_APP_DATABASE_USERNAME=
+DATAROCKETS_FINANCE_APP_DATABASE_PASSWORD=
+
+# Stmp server credentials
+GMAIL_LOGIN=
+GMAIL_PASSWORD=
+
+# Devise secret key
+DEVISE_SECRET_KEY=
```

Comment:
👍

Reply:
@sinpai let's initiate all envs with `''` here, otherwise usage of undefined envs can raise errors